lib/TPP/MapToBatchReduceGEMM.cpp

```diff
@@ -125,6 +196,17 @@ getSlicedOperands(OpBuilder &builder, Location loc, ValueRange localIvs,
   return slicedOperands;
 }
 
+static bool isLinalgVNNILoop(linalg::LinalgOp linalgOp) {
```

Comment:
avoid the if pls. Why is called `isLinalgVNNILoop`?

Reply:
I'll rename this, I called it isLinalgVNNILoop because I check for loop's dimensions to be 7 in the method.

Reply:
I've also updated this check to see if the second operand's source is a VNNI pack operation, this works in tensor semantics only.